Solutions/Digital Shadows/Data Connectors/Digital Shadows/DigitalShadowsConnectorAzureFunction/AS_api.py

```diff
@@ -55,11 +57,11 @@ def post_data(self, body, log_type):
         }
 
         response = requests.post(uri,data=body, headers=headers)
-        print(response.text)
+        
         if (response.status_code >= 200 and response.status_code <= 299):
-            logging.info('Accepted')
+            logger.info('Accepted')
         else:
-            logging.info("Response code: {}".format(response.status_code))
+            response.raise_for_status()
```

Comment:
Should we keep the if-block? We will know if the data got ingested in the logs when it prints 'Accepted'.

Reply:
if you want to log `Accepted`, do this:

```
response.raise_for_status()
log.info('Accepted')
```

You don't need the if block - if you look at the documentation and source for `raise_for_status` it will raise a `HttpError` if there was an error - so the `log.info('Accepted')` only gets logged if there was no error.

https://docs.python-requests.org/en/latest/_modules/requests/models/#Response.raise_for_status